sdk/test-utils/perfstress/build.ps1

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker build -t azure-sdk-for-js-perfstress -f Dockerfile ../../../
```

Comment:
I'm not currently using this file, but it will be relevant for: https://github.com/azure/azure-sdk-for-js/issues/8063